### PR TITLE
improvement(graphs-ux): graphs filters reorganization

### PR DESCRIPTION
**Pull request by soyacz**
Some improvements around graphs filtering: less colors, more organization and labels.
image

### CODE REVIEW

File: frontend/TestRun/ResultsGraphs.svelte [hunk 1]
```diff
@@ -172,48 +173,81 @@
...
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(3, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(3); showCustomInputs = false;}}>
+            Last 3 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {startDate === dayjs().subtract(6, 'month').format('YYYY-MM-DD') ? 'active' : ''}"
+                on:click={() => {setDateRange(6); showCustomInputs = false;}}>
+            Last 6 Months
+        </button>
+        <button class="btn btn-outline-primary btn-sm {showCustomInputs ? 'active' : ''}"
```
Comment (k0machi) [inline frontend/TestRun/ResultsGraphs.svelte:190]: This is a good candidate to utilize Svelte reactivity pattern:

```svelte
<script>

     let dateRange = 3;
     $: setDateRange(dateRange);
</script>

<button class:active={dateRange === 3} on:click={() => dateRange == 3}>Last 3 Months</button>
```

File: frontend/TestRun/ResultsGraphs.svelte [hunk 2]
```diff
@@ -172,48 +173,81 @@
...
-        <button
-                on:click={() => toggleColumnFilter(filter)}
-                class:selected={selectedColumnFilters.some(f => f === filter)}
-                style="background-color: #a3e2cc"
-        >
-            {filter}
+    </div>
+</div>
+
+<label>Tables Filters</label>
```
Comment (k0machi) [inline frontend/TestRun/ResultsGraphs.svelte:211]: `<label>` should not be used like that, use either heading `hN` (e.g. `h4`) or a normal `div`

**Review by k0machi — CHANGES_REQUESTED** [review]

File: frontend/TestRun/ResultsGraphs.svelte [hunk 3]
```diff
@@ -172,48 +173,81 @@
     });
 </script>
 <div class="filters-container">
-    <div class="input-group input-group-sm mx-2">
-        <input type="date" class="form-control date-input" bind:value={startDate} on:change={() => fetchTestResults(test_id)} />
-        <input type="date" class="form-control date-input" bind:value={endDate} on:change={() => fetchTestResults(test_id)} />
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(1)}>Last Month</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(3)}>Last 3 Months</button>
-        <button class="btn btn-info btn-sm" on:click={() => setDateRange(6)}>Last 6 Months</button>
+    <label class="mt-1">Time range:</label>
```
Comment (k0machi) [inline frontend/TestRun/ResultsGraphs.svelte:176]: Ditto

**Comment by fruch** [comment]
I think the table/column filter separation/titles is a bit unclear for the common user. what we mean by column here is, and what we mean by table, is not exactly intuitive 

Maybe "group filters" and "metric filter" ? Or something in that direction?

Also it would be really helpful to have tooltips with a longer description for each button we show there

File: frontend/TestRun/ResultsGraphs.svelte (hunk 2 above)
Comment (soyacz) [reply frontend/TestRun/ResultsGraphs.svelte]: I replaced it with <span>

File: frontend/TestRun/ResultsGraphs.svelte (hunk 3 above)
Comment (soyacz) [reply frontend/TestRun/ResultsGraphs.svelte]: ditto

File: frontend/TestRun/ResultsGraphs.svelte (hunk 1 above)
Comment (soyacz) [reply frontend/TestRun/ResultsGraphs.svelte]: thanks, improved

**Comment by soyacz** [comment]
image

**Review by k0machi — APPROVED** [review]

**Comment by fruch** [comment]
> > I think the table/column filter separation/titles is a bit unclear for the common user. what we mean by column here is, and what we mean by table, is not exactly intuitive
> > Maybe "group filters" and "metric filter" ? Or something in that direction?
> 
> I replaced with just `Filters` and `Metrics` - after clicking it should become more intuitive.

Much better

> 
> > Also it would be really helpful to have tooltips with a longer description for each button we show there
> 
> I validated idea and it doesn't play well with filters. E.g. descriptions: for `read - _mgmt_repair_cli - stalls - REACTOR_STALLED` is `REACTOR_STALLED event counts` or table `read - _mgmt_repair_cli - latencies` has `read workload - Scylla-Manger repair`. I'd rather see descriptions in graphs itself, under the titles.

Logicly I would argue that it's best to be visible when you are selecting what to look at.
and not after browsing all of the graphs.
nevertheless, we should be able to see those descriptions for anything that has a description.

**Review by fruch — APPROVED** [review]
LGTM

**Comment by soyacz** [comment]
> Logicly I would argue that it's best to be visible when you are selecting what to look at. and not after browsing all of the graphs. nevertheless, we should be able to see those descriptions for anything that has a description.

https://github.com/scylladb/argus/issues/496

**Pull request merged**